doc/kernel/services/data_passing/pipes.rst

```diff
@@ -54,8 +54,9 @@ pended writer to fill the ring buffer.
     Flushing does not in practice allocate or use additional buffers.
 
 .. note::
-    The kernel does NOT allow for an ISR to send or receive data to/from a
-    pipe or flush even if it does not attempt to wait for space/data.
+    The kernel does allow for an ISR to flush a pipe from an ISR. It also
```

Comment:
It would be nice to preemptively add this to doc/releases/release-notes-3.2.rst to make sure we don't forget to note the change in the release crush. Not a blocking comment.

Reply:
I'll make a note of that locally. Updating the 3.2 release notes is something that I would like to do after this gets merged (if I may be so presumptuous) to reduce the likelihood of merge conflicts.